src/components/Button.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Button(prop) {
+  return <button type="button">{prop.name}</button>;
+}
+
+export default Button;
```

Comment:
its button name prop should be a String ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png))